challenges/day-01/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Day 01</title>
+
+    <style>
+       
+        body {
+            
+            padding: 0 0 0 10px;
+        }
+      
+        .block {
+            height:100px;
+            width:100px;
+            background-color:#fa7e70;
+            margin: 10px 0 10px 0;
+            
+        }
+        .block2 {
+            height:100px;
+            width:100px;
+            background-color:#fa7e70;
+            margin: 10px 0 10px 0;
+            border-radius: 10px;
+        }
+        .block3 {
+            height:100px;
+            width:100px;
+            background-color:#fa7e70;
+            margin: 10px 0 10px 0;
+            border-radius: 35%;
+        } .block4 {
+            height:100px;
+            width:100px;
+            background-color:#fa7e70;
+            margin: 10px 0 10px 0;
+            border-radius: 50%;
+        }
```

Comment:
This works great, but notice how there is a lot of duplicated code? We have a principle in programming known as the DRY principle, which stands for **D**on't **R**epeat **Y**ourself.

If I had changed my mind and said that I want the blocks to each be 200px, you'd have to change it for every single block here. So here's a proposed alternative:

```suggestion
        .block {
            height:100px;
            width:100px;
            background-color:#fa7e70;
            margin: 10px 0 10px 0;
        }
        .block.smooth {
            border-radius: 10px;
        }
        .block.rounded {
            border-radius: 35%;
        }
        .block.circle {
            border-radius: 50%;
        }
```

Reply:
Thank you so much for that advice. I applied the DRY protocol as per your suggestion and it looks alot cleaner and works better.